packages/components/src/number-input/src/NumberInput.css

```diff
@@ -71,6 +71,9 @@
 
 .o-ui-number-input-spinner .o-ui-icon {
     fill: var(--o-ui-icon-alias-primary);
+    align-self: center;
+    width: 1rem;
```

Comment:
all icons inside a number input are reduced in size to 16x16?

Reply:
looks like a mistake

Reply:
All icons in an input spinner which is the buttons to up down the numbers

Reply:
you're right, my bad